pkg/kubelet/cm/dra/plugin/plugins_store.go

```diff
@@ -88,31 +41,22 @@ func (s *PluginsStore) Get(pluginName string) *Plugin {
 
 // Set lets you save a DRA Plugin to the list and give it a specific name.
 // This method is protected by a mutex.
-func (s *PluginsStore) Set(pluginName string, plugin *Plugin) {
+func (s *pluginsStore) set(pluginName string, p *plugin) {
```

Comment:
nit: this function is an opposite to `delete` as far as I understand. Would it be better to call it `add`?

Reply:
done in 251e6a4f7eb